src/rbcx.pb.h

```diff
@@ -14,77 +14,123 @@ extern "C" {
 #endif
 
 /* Struct definitions */
-typedef struct _SetMotorPower {
-    uint32_t motorId;
-    uint32_t power;
-} SetMotorPower;
+typedef struct _CoprocRequest_ButtonsRequest {
+    char dummy_field;
+} CoprocRequest_ButtonsRequest;
 
-typedef struct _StmMessage {
+typedef struct _CoprocStatus_LedsStatus {
+    char dummy_field;
+} CoprocStatus_LedsStatus;
```

Comment:
Btw, tím, že je ta `message` zanořená, si tady vytváříme hrozně dlouhá jména, chtělo by to zkrátit (třeba odělat jedno z těch `Status`/`Request`).

Reply:
Jj, tak to odnoříme?

Reply:
Podle mě to zanoření není špatné (hned vidíš, co kam patří), ale zkusil bych to zkrátit: Request -> Req, odstranit zdvojené "Status" a pokud tam zrovna není potřeba tak Get (co jinýho bys s tlačítkama dělal...).

Reply:
Zkráceno